presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -45,31 +36,7 @@
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
     private CredentialProviderType credentialProviderType = INLINE;
-    private String typesMappedToVarcharInclude;
-    private String typesMappedToVarcharExclude;
-    private Set<Integer> includeTypesMappingToVarchar;
-    private Set<String> includeTypeNamesMappingToVarchar;
-    private Set<Integer> excludeTypesMappingToVarchar;
-    private Set<String> excludeTypeNamesMappingToVarchar;
-
-    private static List<Field> getStatics(Class<?> clazz)
-    {
-        return Arrays.stream(clazz.getDeclaredFields()).filter(f ->
-                Modifier.isStatic(f.getModifiers())).collect(toList());
-    }
-
-    private static final Map<String, Integer> typesMap = new HashMap<String, Integer>()
-    {
-        {
-            for (Field f : getStatics(Types.class)) {
-                try {
-                    put(f.getName(), f.getInt(f));
-                }
-                catch (Exception e) {
-                }
-            }
-        }
-    };
```

Comment:
please squash the commits. Changes in single PR should not go back and forth.

Reply:
:) sorry @kokosing i didn't warn you this has significantly altered since the beginning

Reply:
@findepi @kokosing Rebased and squashed to one commit 
Will add now the end-to-end test to `TestPostgreSqlTypeMapping `